src/integrations/GA4/browser.js

```diff
@@ -74,13 +87,18 @@ export default class GA4 {
   }
 
   /* utility functions ---Start here ---  */
+
+  /**
+   * If the gtag is successfully initialized, client ID and session ID fields will have valid values for the given GA4 configuration
+   */
   isLoaded() {
-    return !!(window.gtag && window.gtag.push !== Array.prototype.push);
+    return !!(this.clientId && this.sessionId);
```

Comment:
Do we need double logical not !!

Reply:
`!!(this.clientId && this.sessionId);` and `this.clientId && this.sessionId` are exectly same but since we are using` !!` across sdk that's why we kept it